### PR TITLE
Feature/dc 755 add task app

### PR DESCRIPTION
**Pull request by oleksii-symon-corva-ai**
[DC-755 Add task app type to Corva](https://corvaqa.atlassian.net/browse/DC-755)

- Added task app type to Corva
- simplified patching in plugin._patch_scheduled

### CODE REVIEW

File: src/corva/runners/task.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    api: Api, task_id: str, status: TaskStatus, data: dict
+) -> requests.Response:
+    """Updates the task. Should never raise."""
+
+    return api.put(path=f'v2/tasks/{task_id}/{status}', data=data)
+
+
+def task_runner(fn: Callable, context: TaskContext):
+    try:
+        # TODO: in case this raises, should exc be added to fail_reason?
```
Comment (oleksii-symon-corva-ai) [inline src/corva/runners/task.py:26]: @ville what do you think about this?

Comment (ville) [reply src/corva/runners/task.py:26]: If this throws an error, then most likely update_task_data will fail as well.

**Review by ville — APPROVED** [review]
LGTM

File: src/corva/runners/task.py (hunk 1 above)
Comment (oleksii-symon-corva-ai) [reply src/corva/runners/task.py:26]: Yes. `update_task_data` doesnt raise, so the task will just silently finish without updating the task. So the task will stay in running state

**Pull request merged**